modules/planning/scenarios/util/util.cc

```diff
@@ -255,8 +255,13 @@ bool CheckADCReadyToCruise(Frame* frame,
   ADEBUG << "heading_align_w_reference_line: "
          << heading_align_w_reference_line;
   // check gear status
+  // TODO(SHU): align with vehicle parameters
+  static constexpr double kMinSpeed = 0.1;  // m/s
```

Comment:
Do you plan to enable Cruise when GAER is possibly in `Neutral` or `Reverse`? Otherwise why change the `&&` to `||` ?

Reply:
There is constraint for stop speed.